src/components/CompareResults/ResultsMain.tsx

```diff
@@ -33,17 +36,35 @@ function ResultsMain(props: { results: CompareResultsItem[][] }) {
     }),
   };
 
+  const filteredResults = useMemo(
```

Comment:
Honestly I wasn't sure whether to put that here or in ResultsTable. I decided to put it here so that I wouldn't have to move the searchTerm down the component chain, but this is debatable. We could filter _and_ process in `processResult` so that we'd loop the results once less.

Also `searchTerm` could be in redux in the future, which for example would make it easier to highlight the search term in the results.

But I decided to keep it simple for now.